deployment/permissions/service.quiz.quizuserentry.ini

```diff
@@ -0,0 +1,5 @@
+[action_permission_items]
```

Comment:
I don't think this file is needed, the permissions already granted in service.userentry.ini.